repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/arrays/ArrayComparator.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.openrepose.commons.utils.arrays;
 
+@FunctionalInterface
```

Comment:
Actually, it's a completely crap interface that makes no sense now that I take a closer look.  Nothing uses it except for the one class that implements it.  I'm going to delete the whole interface.